packages/services/api/src/modules/organization/module.graphql.ts

```diff
@@ -39,6 +66,24 @@ export default gql`
     Remove this mutation after migration is complete.
     """
     migrateUnassignedMembers(input: MigrateUnassignedMembersInput!): MigrateUnassignedMembersResult!
+    exportOrganizationAuditLog(
+      selector: OrganizationSelectorInput!
+      filter: AuditLogFilter
+      pagination: AuditLogPaginationFilter
+    ): ExportOrganizationAuditLogResult!
```

Comment:
TODO @TuvalSimha This field should be in `modules/audit-logs/module.graphql.ts`, same for the implemnted resolver.